src/hooks/chat/useBodyScrollControllerEffect.tsx

```diff
@@ -44,6 +44,17 @@ export default (
     chatBodyRef.current.scrollTop = scrollTop;
   }, [chats]);
 
+  const lastChat = chats.length > 0 ? chats[chats.length - 1] : {};
+  useEffect(() => {
+    socketReady(() =>
+      axios({
+        url: "/chats/read",
+        method: "post",
+        data: { roomId },
+      })
+    );
+  }, ["time" in lastChat ? lastChat?.time : ""]);
```

Comment:
```suggestion
  }, [lastChat?.time, roodId]);
```

과 차이가 없을 것 같습니다 ! 그리고 `roodId` 도 추가되어야 할 것 같아요.